app/jobs/itms_job.rb

```diff
@@ -28,36 +28,48 @@ def self.get(url)
       raise DownloadError, "Unexpected response code: #{response.code} for url: #{url}" if response.code != 200
       response
     end
+
+    private
+
+    def self.build_href(href, params)
+      params.reject! { |_, v| v.blank? }
+      URI(href).tap { |url| url.query = [url.query, params.to_query].compact.join('&') }.to_s
```

Comment:
volal by som radsej `#reject` bez `!` lebo na tom sa mozes popalit, kludne takto:

```suggestion
      URI(href).tap { |url| url.query = [url.query, params.reject { |_, v| v.blank? }.to_query].compact.join('&') }.to_s
```